### PR TITLE
embed qc list in files

### PR DESCRIPTION
**Pull request by KorayKirli**
Extending the embedded list for files with qclist, to be used by qc related pipeline checks on foursight.

### CODE REVIEW

**Comment by j1z0** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=5411760&bid=17742124)

File: src/encoded/types/file.py [hunk 1]
```diff
@@ -1290,20 +1291,20 @@ def download(context, request):
...
+            "ea": "Range Query" if request.range else "File Download",  # Event Action
             "el": filename,                         # Event Label
             "ev": file_size_downloaded,             # Event Value
             # Product fields
             "pa": "purchase",
             "ti": str(uuid4()),                     # We need to send a unique transaction id along w. 'transactions' like purchases
             "pr1id": file_at_id,                    # Product ID/SKU
             "pr1nm": filename,                      # Product Name
-            "pr1br" : lab.get("display_title"),     # Product Branch
+            "pr1br": ab.get("display_title"),     # Product Branch
```
Comment (sbreiff) [inline src/encoded/types/file.py:1302]: Was this intentional? lab --> ab?

Comment (andreacosolo) [inline src/encoded/types/file.py:1302]:
```suggestion
            "pr1br": lab.get("display_title"),     # Product Branch
```
I have no idea of the context, but sounds like "lab" potentially made sense?

Comment (KorayKirli) [reply src/encoded/types/file.py:1302]: definitely not, will get it back to lab nice catch

**Comment by j1z0** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=5411760&bid=17742124)

**Pull request merged**